L1Trigger/DTTrigger/src/DTTrigTest.cc

```diff
@@ -194,11 +194,12 @@ void DTTrigTest::beginJob() {
 }
 
 void DTTrigTest::beginRun(const edm::Run& iRun, const edm::EventSetup& iEventSetup) {
-  if (!my_trig) {
-    my_trig->createTUs(iEventSetup);
-    if (my_debug)
-      cout << "[DTTrigTest] TU's Created" << endl;
-  }
+  if (!my_trig)
+    return;
```

Comment:
@cms-sw/l1-l2 , I think after https://github.com/cms-sw/cmssw/commit/ebbb7e33380fe684ce3d9558b782f1c300ba66f0#diff-72571f83f22091a91c0533331585af8bb25f0031cf267c6ad8e5bb44241ee72e change there is no need to have this check ... `my_trig` is created in the constructor